<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiagChangeInMultipleFields</class>
 <widget class="QDialog" name="DiagChangeInMultipleFields">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cambio de valor a un campo de productos elegidos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Seleccione en que campo desea realizar el cambio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbCampo"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nuevo valor para el campo seleccionado</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leValueText"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leValueDouble"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leValueInt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Seleccione que tipo de modifcación realizar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbModType">
         <item>
          <property name="text">
           <string>Mismo valor en todos los productos</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fórmula</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Si el campo elegido es numérico puede definir una operación matemática a realizar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leFormula"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
